CMakeLists.txt

```diff
@@ -246,8 +280,12 @@ mark_as_advanced(GOOGLE_CLOUD_CPP_ENABLE_EXAMPLES)
 
 add_subdirectory(google/cloud)
 foreach (library ${GOOGLE_CLOUD_CPP_ENABLE})
-    if (${library} STREQUAL generator)
+    if ("${library}" STREQUAL generator)
```

Comment:
we put quotes around "experimental-storage-grpc". do we want "generator"?

Reply:
Done.